src/hooks/screen/useChatScreenHook.ts

```diff
@@ -184,7 +208,8 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
       }
 
       await currentChatSchema.updateChatSentStatus(localDb, response);
-      refresh('chat');
+      refreshWithId('chat', selectedChannel?.id);
+      // refresh('chat');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, the `refreshWithId` function is used instead of the `refresh` function. The old `refresh` call should be removed.

```diff
      refreshWithId('chat', selectedChannel?.id);
-212:       // refresh('chat');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->